### PR TITLE
Bomb

### PR DESCRIPTION
**Pull request by yoysam**
### Purpose
fixes the firist part of bomb #4676 

### Notes:
added a simple loop

### CODE REVIEW

**Comment by VernaSaurus** [comment]
bomb

File: UnityProject/Assets/Scripts/Explosions/ExplosionNode.cs [hunk 1]
```diff
@@ -28,10 +32,10 @@ public void Process()
 			float EnergyExpended = 0;
 			var v3int = new Vector3Int(Location.x, Location.y, 0);
 
-			EnergyExpended = matrix.MetaTileMap.ApplyDamage(v3int, AngleAndIntensity.magnitude,
-				MatrixManager.LocalToWorldInt(v3int, matrix.MatrixInfo), AttackType.Bomb)*0.375f;
+			EnergyExpended = matrix.MetaTileMap.ApplyDamage(v3int, DD,
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Explosions/ExplosionNode.cs]: might as well rename DD to something better

Comment (yoysam) [reply UnityProject/Assets/Scripts/Explosions/ExplosionNode.cs]: did it

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Explosions/ExplosionNode.cs]: cool

**Comment by waffielz** [comment]
bomb

**Pull request merged**